src/libraries/System.Runtime/tests/System/TupleTests.cs

```diff
@@ -428,7 +428,7 @@ public static void CompareTo()
             tupleDriverB = new TupleTestDriver<short, int, long, string, char, float, double, DateTime, Tuple<bool, object>, TimeSpan>((short)1, (int)1, long.MinValue, "This");
             tupleDriverC = new TupleTestDriver<short, int, long, string, char, float, double, DateTime, Tuple<bool, object>, TimeSpan>((short)1, (int)1, long.MinValue, "this");
             tupleDriverA.CompareTo(tupleDriverB, 0, 5);
-            tupleDriverA.CompareTo(tupleDriverC, 1, 5);
+            tupleDriverA.CompareTo(tupleDriverC, PlatformDetection.IsInvariantGlobalization ? /* 'T'-'t' */ -32 : 1, 5);
```

Comment:
Is this correct? The sorting flips on invariant globalization?

Reply:
Yeah, because `Invariant` compares the chars directly by doing a substraction of them:

https://source.dot.net/#System.Private.CoreLib/SpanHelpers.Char.cs,114